packages/plugins/table/src/index.tsx

```diff
@@ -2,26 +2,27 @@ import { StatefulPlugin, string } from '@edtr-io/plugin'
 import * as React from 'react'
 import ReactMarkdown from 'react-markdown'
 
-import { createTableEditor } from './editor'
+import { TableEditor } from './editor'
 
 export const tableState = string()
 
 export function createTablePlugin(
-  config: TablePluginConfig
-): StatefulPlugin<typeof tableState> {
+  renderMarkdown = defaultRenderMarkdown
```

Comment:
A plugin exposes an arbitrary factory (convention: named `createFooPlugin`). Here, feel free to add default values for optional stuff.